frontend/src/pages/Player/PlayerHook/PlayerHook.tsx

```diff
@@ -295,7 +283,7 @@ export const usePlayer = (
 			}
 			return toRemove
 		},
-		[getChunkTs, noChunkRemoval],
```

Comment:
We said this flag could be cleaned up and rolled out for everyone.

Reply:
sounds good!